src/UserCounterBundle/Resources/config/doctrine/User.orm.yml

```diff
@@ -0,0 +1,24 @@
+UserCounterBundle\Entity\User:
+    type: entity
+    table: null
+    id:
+        id:
+            type: integer
+            id: true
+            generator:
+                strategy: AUTO
+    fields:
+        email:
+            type: string
+            length: 255
+            unique: true
+        dob:
+            type: date
+        accountNumber:
```

Comment:
missing column: account_name otherwise DB will have the field name accountNumber